test/notifications/display_test.dart

```diff
@@ -169,11 +169,20 @@ void main() {
         expectedTagComponent: 'stream:${message.streamId}:${message.subject}');
     }));
 
-    test('group DM', () => awaitFakeAsync((async) async {
+    test('group DM: 3 users', () => awaitFakeAsync((async) async {
       await init();
       final message = eg.dmMessage(from: eg.thirdUser, to: [eg.otherUser, eg.selfUser]);
       await checkNotifications(async, messageFcmMessage(message),
-        expectedTitle: "${eg.thirdUser.fullName} to you and 1 others",
+        expectedTitle: "${eg.thirdUser.fullName} to you and 1 other",
+        expectedTagComponent: 'dm:${message.allRecipientIds.join(",")}');
+    }));
+
+    test('group DM: more than 3 users', () => awaitFakeAsync((async) async {
```

Comment:
Should I label this test as '4 users' rather than 'more than 3 users' or use another label for the test? My idea for this test is to test the plural form of the string when there are more than 3 users in the conversation that is why I labeled it as such.

Reply:
Yeah, that reasoning sounds good to me. The important thing for the test to do its job is that there are more than 3 users; having exactly 4 is just a detail of how the test happens to be implemented.